src/test/resources/ldap/elytron-keystore-tests.ldif

```diff
@@ -162,118 +162,152 @@ userPKCS12:: MIISsgIBAzCCEmwGCSqGSIb3DQEHAaCCEl0EghJZMIISVTCCBWoGCSqGSIb3DQEHA
- c6Fk09crMO70bRQGig6HWLPA+os+viwqQqEu78g1t2bGDqjzJLhN9ht995AGpNbWgUAeABPEau1L5
- hISB7NCJDS6j8/5beHrI/23gkISR7CtH1aJacvtbA49xBuceZ7WMPoaUNmmuStZOL8RZakgUh4Msi
- kq3x7UiBhGdFfynQU93jhh4ZX7N3G54dJJ7QDwwGg+VtnxFXbwgFMXIDhhd6JUirxwyhncn2PC0pk
- VkVmt0oDWAL9AIPC+qzpkFb0ut+McJlT7kJhIeJ1cU2ttQ9QeC5cASeU7Zx/rRBDgWCtHiBgTE3Nr
- a0FQITaLEGIxxMzwmbYHIf7BzuT3wpLNrCM7WXC7UG/b78LJmRpL9V2Ihqi0Un5F+jh6kLBFa+2C6
- Z68j2dGvZ7X28aBk1fNhha8bpk+K5JLvKIALdSf3ISSxVEpA5ks1fzBpVxbvhkInvWETTwACO4tEc
- bXZ88bbd9NnSGg/q+yOfTn9cVXHz/sKck05Ft8v3MfgqtPJE9IU5nU4BmiOcfojppxoje6373cqzh
- 6MUyOV9ZxRhtUc50kYWJUshKJp7EDcdRQn2Ue+QuDsjSqzJfkraETFrHBMiwuazAHuJyjNowmVDIi
- nK231QCF2iqTKrnNQUJNjJlFfmc/0NGDj+euIRxpFD5yOS6VPmu50RJ2qYJ8WZ1NqRKraOa8DMpAf
- oiGaYluEMxOx7jWlPT1yu0eg1iG0J3QoQXCT7BVjTbJ+AC5Z85N+aFfwMTTYk/XgvWBHJ3SLZfKOm
- lhd+eJu7J+Dh0cXQwAmsuEyTnSz9uTUr1h1aZtKHY76bzcRrjYUREXXeCprypSHF07sLb1FhO132K
- CYIiVLizZLlRhbeFNsZL8CA4aFEB2vAaAcdieVqN6SMPfcEfjUyGmsonSRa5V+HmxZ9A2hB6ftZ3w
- 4uUwrULiI8RxoNg53vPhrGTOyit1USUot86hr2Cgbycfxpx+KEo+KPA8PWnflOY0w8Kzuzwd+3fFs
- wOE+4DBS4zIx0bff32CpviM2cW9iYF1fhPtJwIW35QwkkqCCCQf6holMOkZUbtTeILCptFE3dqr+0
- Q+MXBWFIUyIQS9399puXKrWRFqGae7+Q07EIlZy59wzSmGWvzqIdFkC1KgZZYkhoFibf4ODtYl0Ti
- 3H+6xIaq0biiMLD9sJYXjPQVBBMFsP50sExiGdDuHic2fYXy0ghMSR2HMT4wGQYJKoZIhvcNAQkUM
- QweCgBtAHkAawBlAHkwIQYJKoZIhvcNAQkVMRQEElRpbWUgMTUzMzAyMjA4NTY3MDCCBEsGCSqGSI
- b3DQEHBqCCBDwwggQ4AgEAMIIEMQYJKoZIhvcNAQcBMCgGCiqGSIb3DQEMAQYwGgQU+hsi2rsMIlu
- 971aewS53dATb0rQCAgQAgIID+Hy19DB32fS9uz7jc+DIMwvSQO4jm9J+iMa8wMDZa7bUGrXkn/RZ
- 7HgdMXx4d5ghWeEghSB5HBBcmEgSA9EbMFBec++g8vQtZiP3R0EARNQLrYecZDbiEcW4/PQhaHWp3
- U72GGe4v/QfChbwW8XhfzySQeHQWodg+uSCA32mGpjxS7NtqTHXIbVK4/izEiwKrEqQF8G/5aVfz8
- nLZ46KUaPGIYp42IUpaLEiqW+JPFFZjOWQz2NiEuz1eYM4hNNDTMcEEUfumtawBIjC2H/k26WTdH3
- 3EImjWi1ie35Rl6nkwHHu0zTYbuQJm3IDJpHpZaXsg/4auEr3MieNUITxB7WBOt7NKFI5/PGBf/7f
- n/M+Lj+vPcy01pdglNv8Vgr3ZwIhDUe3OOvpb/3eyjIte+ywOMzLaTHfW+5mvCX/vZFeGx9FLMlFe
- 4SehRt6slMcipijF866QQoYZ3svvsUguQCHP3Vnq4Hl9PLCYmWyHEKgiUP+YD7q9JWWpEHN2nUNDY
- NwolzQpVHXubip1gbIuWzMeoO9xP84sB/vyinCz0ZcRDNSefQ9/ieNNoAHu9TsigFzd0SFBnktWNs
- nbw63Ex6C2nNhybjlmSY74UzoqAcjmm9cZA+Iu+aiuRKbVBP0DFgPeQbNugww4SQjy+LRFGQE+Zxh
- +L2TXUMoDuyCxBnriIASb4oNKU2Yd4wSU90KE6IDn7YkO8rXXU0IPN81AUuvILpEyvGx2xjvfFOjq
- iBMVaOcnZ/BpVA0RHsTk8l1Dqw4aNy1tYSfaWw+uJjjG6PhulRWgabAtLdX85P3dBd1rkyVflaTQY
- ykm+ZqeMWz56MCa7swE3uYEvDD+tuFYUzCjThqgzIbCDT6g42x5k9FMNylrHdY2fUvDhMugZRu3RP
- 1GT8cz9SYtA8yX/jwPgGWh+f4O7j24Zm29QXR++/j6rXnm/53tOk+OsaA4FGm29OEEQbD30UFsslq
- +GiCMWyrbhO1+G8qeA31jeDrPReUlEcjpnoNfGr0yw1fyUk8PBX2snHXwOfGAKOOYTbtRTooiVLcr
- CbmVCSDVTU+Y/D72z69K3Ff/+wuKOn2k/1lMZKr7jWU9C6TOL/goIZ3gAxBrhJXDvAhDxTK44G491
- +HVaTufREUuGQ44wMLqCBSLrf4yTPxocrKeHk/rfP9eeqWjh7TRUy7fETP7+j6pTdIbypeTsODFnG
- 6s+kpHeZ5bLByFJmJVXN+vZ/ihbiJyPgEVnMAyepQJY8qcUaNP/j8CTpH076l+ncKku4EdXtb65Lo
- he0Efo1JsZF5J98EVOwi/36ROfYSHdu3WAKbyPSCPAlWNem1mH5Cv2yI4k7PYSdIocM+T8VPMD0wI
- TAJBgUrDgMCGgUABBQDPZHExnAQLijTT67kcr3VPwYNugQU0zGDk7T4piqiiJsVYLNzDQ9eZYgCAg
- QA
+cn: binary
```

Comment:
Thanks for working on this one @JiriOndrusek. Would it be possible to add some comments on how these were generated for future reference?

Reply:
@fjuma I'm not sure if this is necessary, because all certificates was generated according https://github.com/wildfly-security/wildfly-elytron/blob/master/src/test/resources/ca/Readme.txt. And directly in this ldif file is comment describing it: https://github.com/wildfly-security/wildfly-elytron/blob/master/src/test/resources/ldap/elytron-keystore-tests.ldif#L1.

Unfortunately I missed it during creation of this test (with certificates generated differently).

Reply:
Perhaps it would be good to add a comment at the top of ```KeyStoreSuiteChild``` that mentions that the certificates and ldif entries for the tests in this class were generated using the files you referenced.

Reply:
@fjuma It seems reasonable, I'll do it.